util/io_test.go

```diff
@@ -217,16 +217,23 @@ func TestSanitizeArchivePath(t *testing.T) {
 func TestListFilesInDir(t *testing.T) {
 	tmpDir := TempDirPath()
 
-	file1, err := os.Create(filepath.Join(tmpDir, ".file1"))
+	file1Path := filepath.Join(tmpDir, ".public_file")
```

Comment:
Opps, we should not have dot here. @Ja7ad please review carefully. Let's fix this test again

Reply:
revert it?

Reply:
No, new PR. Only remove dot from public file

Reply:
https://github.com/pactus-project/pactus/pull/1566